src/sql/azdata.d.ts

```diff
@@ -4314,6 +4314,11 @@ declare module 'azdata' {
 			 * @return The given range or a new, adjusted range.
 			 */
 			validateCellRange(range: CellRange): CellRange;
+
+			/**
+			 * Sets the notebook to trusted by default.
+			 */
+			setTrusted(state: boolean);
```

Comment:
This should go into https://github.com/microsoft/azuredatastudio/blob/master/src/sql/azdata.proposed.d.ts

(just define NotebookDocument in there - the two definitions are combined during build time)